Sources/_NIO1APIShims/NIO1APIShims.swift

```diff
@@ -522,6 +522,9 @@ public typealias HTTPUpgradeEvents = HTTPServerUpgradeEvents
 @available(*, deprecated, renamed: "HTTPServerUpgradeErrors")
 public typealias HTTPUpgradeErrors = HTTPServerUpgradeErrors
 
+@available(*, deprecated, renamed: "NIOWebSocketServerUpgrader")
+public typealias WebSocketUpgrader = NIOWebSocketServerUpgrader
```

Comment:
This can't go in the `_NIO1APIShims` module as it's going away. This typealias will need to go into the `NIOWebSocketServerUpgrader` file.

Reply:
Okay, thanks. Have moved it now.